src/arrays/arrays_with_methods/hard/q6/MainTest.java

```diff
@@ -0,0 +1,73 @@
+package arrays.arrays_with_methods.hard.q6;
+
+import global.MethodTest;
+import global.utils.ArrayUtil;
+import global.variables.clauses.RandomInteger;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class MainTest {
+    // Java
+
+    public static Stream<Integer> randomReplaceInput(){
+        return Stream.of(3, 4, 5, 6, 7, 50);
+    }
+
+    @ParameterizedTest
+    @MethodSource("randomReplaceInput")
+    public void correctRandomReplaceMethod(int size) throws Throwable {
```

Comment:
You don't need this return type, the custom assertion does this for you.

Reply:
we have a getMethodReturnType